### ISSUE TITLE
Can't access meta data of file

### ISSUE THREAD

**Issue by Otto-AA** [solid/node-solid-server#1380]
After successfully creating a .meta file, I cannot access it. Requesting it returns a 404 response with the message "Trying to access metadata file as regular file". You can test it here: https://otman.solid.community/public/revive.jpeg.meta

The error is thrown in headers.js->linksHandler(...), but I don't know what this check is for: https://github.com/solid/node-solid-server/blob/19d1bf0ff5a9a59bb59300b8fce3bfcd038d0ea7/lib/header.js#L60-L64

Doing the same for a container works: https://otman.solid.community/public/.meta

**Comment by jaxoncreed** [comment]
The solid spec has no concept of '.meta' files. It's just something that NSS creates to keep track of meta information. It shouldn't be fetched like a regular file. The contents of a meta file are transferred into the response headers of its corresponding file.

**Comment by Otto-AA** [comment]
I have to disagree with this, because the [content-representation](https://github.com/solid/solid-spec/blob/master/content-representation.md#metadata) part of the spec has its own Metadata section which speaks exactly about this and even uses .meta as an example extension.

It also states that "[...] it [metadata resources] can still be modified by client apps using the methods described in this section", which imo suggests that we can do GET/PUT/... requests on them.

**Comment by kjetilk** [comment]
Note that there is quite a big work around these mechanisms now: https://github.com/solid/data-interoperability-panel/blob/resource-metadata/resource-metadata/hypothesis.md
